plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts

```diff
@@ -17,18 +17,24 @@
 import { createTemplateAction } from '@backstage/plugin-scaffolder-node';
 import { InputError } from '@backstage/errors';
 import { Config } from '@backstage/config';
+import { examples } from './createProject.examples';
+import { FetchApi } from '@backstage/core-plugin-api';
 
 /**
- * Creates the `sentry:craete-project` Scaffolder action.
+ * Creates the `sentry:create-project` Scaffolder action.
  *
  * @remarks
  *
  * See {@link https://backstage.io/docs/features/software-templates/writing-custom-actions}.
  *
  * @param options - Configuration of the Sentry API.
+ * @param fetchApi - fetch implementation to use for calling Sentry service.
  * @public
  */
-export function createSentryCreateProjectAction(options: { config: Config }) {
+export function createSentryCreateProjectAction(
+  options: { config: Config },
+  fetchApi?: FetchApi,
```

Comment:
as suggested elsewhere, you should probably remove this change to to the creation function itself for now